scaladoc/src/dotty/tools/scaladoc/snippets/WrappedSnippet.scala

```diff
@@ -43,7 +43,7 @@ object WrappedSnippet:
       outerLineOffset,
       outerColumnOffset,
       notEmptyClassInfos.size + notEmptyClassInfos.flatMap(_.names).size + packageName.size,
-      notEmptyClassInfos.size * 2 + 2
+      notEmptyClassInfos.size * 2
```

Comment:
same, wy do we need to mutliply by 2 ?

Reply:
It's because we use there indent of lenght 2, so each level of indentation generates 2 spaces